compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.test.cpp

```diff
@@ -136,6 +136,20 @@ class TransposeReshapeTransposeGraph : public TestIOGraph
   luci::CircleConst *_back_perm = nullptr;
 };
 
+bool is_transpose_removed(loco::Graph *g)
+{
+  bool is_transpose_removed = true;
```

Comment:
This name is duplicated with function name. How about changing the name to `ret`?

Reply:
this function searches Transpose Op.
maybe we can use "transpose_exist" and return "not transpose_exist"

Reply:
Anyway, it would be better to avoid duplication of names.

Reply:
Thanks for pointing out. 
updated here - https://github.com/Samsung/ONE/pull/12077/commits/51002fe1eedcf578971e30bcca9603ca966adf35